main.go

```diff
@@ -32,30 +33,44 @@ var (
 	blue   = color.New(color.FgBlue).SprintFunc()
 )
 
-func main() {
-	config := flag.String("c", "", "config path")
-	name := flag.String("n", "", "name of specific test to run")
-
-	var envs pkg.KeyValuePairs
-	flag.Var(&envs, "e", "environment variable")
+func init() {
+	log.SetFlags(0)
+}
 
-	flag.Parse()
+func main() {
+	// prepare the environment
+	var configPath string
+	var testByName string
+	var eVariables pkg.KeyValuePairs
+
+	rootCmd := cobra.Command{
+		Use:   "litmus",
```

Comment:
I haven't run this yet but would having the root command as litmus mean the application has to be run as `litmus litmus`?

Reply:
No, just `litmus` is fine!
This is perhaps not needed, but just for consistency since I would add it for subcommands, I also did so here 👍

Reply:
Makes sense too!